Example/Backpack/View Controllers/BPKRootListTableViewController.m

```diff
@@ -17,13 +17,27 @@
  */
 
 #import "BPKRootListTableViewController.h"
+#import <Backpack/Button.h>
+#import <Backpack/Icon.h>
 
 @interface BPKRootListTableViewController ()
+@property (weak, nonatomic) IBOutlet BPKButton *settingsButton;
```

Comment:
I am not sure how I feel about this being a custom button when we could just use as regular `UIBarButtonItem` and `- (instancetype)initWithImage:(UIImage *)image style:(UIBarButtonItemStyle)style target:(id)target action:(SEL)action;`

Reply:
I've now swapped the settings and close buttons out for system ones. Let me know what you think of it now 😃